src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -24,7 +24,7 @@ tile.natura.overworld_logs2.willow.name=Weidenholz
 tile.natura.overworld_logs2.willow.tooltip=Der Weinende
 tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
 tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
-tile.natura.overworld_logs2.sakura.name=Blütenkirschenholz
+tile.natura.overworld_logs2.sakura.name=Blütenkirschholz
```

Comment:
This should have been applied for _all_ references to sakura further down, too.

Reply:
ofc, sorry.